packages/shared/src/models/IPlaygroundMessage.ts

```diff
@@ -81,6 +90,10 @@ export function isAssistantChat(msg: Message): msg is AssistantChat {
   return isChatMessage(msg) && msg.role === 'assistant';
 }
 
+export function isAssistantToolCall(msg: Message): msg is AssistantChat {
+  return isAssistantChat(msg) && (msg.content as ToolCall)?.type === 'tool-call';
+}
```

Comment:
follow up to https://github.com/containers/podman-desktop-extension-ai-lab/pull/2949/files#r2073488474, we should now have a single type/interface with several meaning.

AssistantChat should have an optional `tools` property,